docs/build/html/importTools.html

```diff
@@ -203,19 +203,21 @@
 
 <dl class="py function">
 <dt id="cranetoolbox.importTools.transform.lighten_tweet">
-<code class="sig-prename descclassname">cranetoolbox.importTools.transform.</code><code class="sig-name descname">lighten_tweet</code><span class="sig-paren">(</span><em class="sig-param">tweet: dict) -&gt; (&lt;class 'str'&gt;</em>, <em class="sig-param">&lt;class 'str'&gt;</em>, <em class="sig-param">&lt;class 'str'&gt;</em><span class="sig-paren">)</span><a class="headerlink" href="#cranetoolbox.importTools.transform.lighten_tweet" title="Permalink to this definition">¶</a></dt>
+<code class="sig-prename descclassname">cranetoolbox.importTools.transform.</code><code class="sig-name descname">lighten_tweet</code><span class="sig-paren">(</span><em class="sig-param">tweet: dict</em>, <em class="sig-param">text_name</em>, <em class="sig-param">id_name</em>, <em class="sig-param">date_name) -&gt; (&lt;class 'str'&gt;</em>, <em class="sig-param">&lt;class 'str'&gt;</em>, <em class="sig-param">&lt;class 'str'&gt;</em><span class="sig-paren">)</span><a class="headerlink" href="#cranetoolbox.importTools.transform.lighten_tweet" title="Permalink to this definition">¶</a></dt>
 <dd><p>Lighten a tweet by returning only the fields required for analysis.</p>
 <dl class="field-list simple">
 <dt class="field-odd">Parameters</dt>
 <dd class="field-odd"><p><strong>tweet</strong> (<a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#dict" title="(in Python v3.8)"><em>dict</em></a>) – A parsed JSON tweet</p>
 </dd>
-<dt class="field-even">Returns</dt>
-<dd class="field-even"><p>A tuple of the three values scraped from the passed tweet</p>
-</dd>
-<dt class="field-odd">Return type</dt>
-<dd class="field-odd"><p><a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#tuple" title="(in Python v3.8)">tuple</a>(<a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#str" title="(in Python v3.8)">str</a>, <a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#str" title="(in Python v3.8)">str</a>, <a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#str" title="(in Python v3.8)">str</a>)</p>
-</dd>
 </dl>
+<p>:param text_name : A string to different text field name
```

Comment:
I think something went wrong with your docstrings, it looks like sphinx didn't recognise the fields

Reply:
![Screenshot from 2020-10-05 21-50-56](https://user-images.githubusercontent.com/7062546/95125256-f68cc500-0754-11eb-90ca-ad86a509e38b.png)

Reply:
Ahah! I think it's because you added a space after the name of the param. Finicky thing...

Reply:
I will take a look.